src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -140,6 +140,26 @@
 
     :else (rf/dispatch [:wallet/stop-and-clean-suggested-routes])))
 
+(defn- get-fee-formatted
+  [route]
+  (when-let [native-currency-symbol (-> route first :from :native-currency-symbol)]
+    (rf/sub [:wallet/wallet-send-fee-fiat-formatted native-currency-symbol])))
+
+(defn- insufficient-asset-amount?
+  [{:keys [token-symbol owned-eth-token input-state no-routes-found? limit-exceeded?
+           sender-network-values enough-assets?]}]
+  (let [eth-selected?              (= token-symbol (string/upper-case constants/mainnet-short-name))
+        zero-owned-eth?            (money/equal-to (:total-balance owned-eth-token) 0)
+        input-at-max-owned-amount? (money/equal-to
+                                    (controlled-input/value-bn input-state)
+                                    (controlled-input/upper-limit-bn input-state))
+        exceeded-input             (if eth-selected?
+                                     input-at-max-owned-amount?
+                                     zero-owned-eth?)]
+    (and (or no-routes-found? limit-exceeded?)
```

Comment:
I think we can join two `(or)` in one here without losing any logic

Reply:
Hi @vkjr 

Thanks for the comment!

The expression is:
```clojure
(and (or no-routes-found? limit-exceeded?)
         (seq sender-network-values)
         (or exceeded-input? (not enough-assets?)))
```

I understand you are suggesting to do:
```clojure
(and (or no-routes-found?
         limit-exceeded?
         exceeded-input?
         (not enough-assets?))
     (seq sender-network-values))
```

But it doesn't is logically equivalent, if we rename the vars as `v1`, `v2`, `v3` and `v4`, (to make them easier to understand), this:

```clojure
(and (or v1 v2)
     (or v3 v4))
```

is not the same as:
```clojure
;; With `or` unified:
(and (or v1 v2 v3 v4))

;; Same as:
(or v1 v2 v3 v4)
```

for instance:
```clojure
(let [v1   false
      v2   false
      v3   true
      v4   true
      ex-1 (and (or v1 v2)
                (or v3 v4))
      ex-2 (or v1 v2 v3 v4)]
  (= ex-1 ex-2))
  
;; => false
```